core/src/test/scala/unit/kafka/server/GroupCoordinatorBaseRequestTest.scala

```diff
@@ -606,7 +608,7 @@ class GroupCoordinatorBaseRequestTest(cluster: ClusterInstance) {
 
   protected def shareGroupHeartbeat(
     groupId: String,
-    memberId: String = "",
+    memberId: String = Uuid.randomUuid.toString,
```

Comment:
Since `ShareGroupHeartbeat RPC` is still under development, I think it's acceptable to change the default value from an empty string to a UUID.

Reply:
Yeah, I'm keeping an eye on all this and will make sure KIP-932 incorporates all of the latest refinements before it is completed.

Reply:
Thank you 😺